Babylon/commands/api/organizations/security/update.py

```diff
@@ -22,33 +18,24 @@
 @wrapcontext()
 @timing_decorator
 @pass_azure_token("csm_api")
-@argument(
-    "security_file",
-    type=Path(path_type=pathlib.Path),
+@output_to_file
+@option(
+    "--role",
+    "role",
+    type=str,
     required=True,
+    default="viewer",
+    help="Role RBAC",
 )
-@output_to_file
-@inject_context_with_resource({'api': ['organization_id', 'url']})
-def update(context: Any, azure_token: str, security_file: pathlib.Path) -> CommandResponse:
+@option("--email", "email", type=str, required=True, help="Email valid")
+@argument("id", type=str)
```

Comment:
I can't see the difference between email and id, isn't the same thing ?

Reply:
id is an email too, we can rename it to `identity_id`